docs/source/tools/visual.rst

```diff
@@ -28,3 +28,28 @@ Running the above should produce an image which looks something like this:
 .. image:: images/quickstart.png
   :width: 30%
   :align: center
+
+
+
+Visualizing DAML Contracts - Within IDE
+#######################################
+
+You can generate visual graphs from VS Code IDE. Open the daml project in VS Code and use `command palette <https://code.visualstudio.com/docs/getstarted/tips-and-tricks#_command-palette/>`_. Should reveal a new window pane with dot image. Also visual generates only the currently open daml file and its imports.
+
+Note: You will need to install the Graphviz/dot packages as mentioned above.
+
+
+Visualizing DAML Contracts - D3 Network
```

Comment:
Might be worth pointing out what `D3` is or avoid mentioning it altogether in favor of something like “interactive graphs”.

Reply:
Using interactive graphs and links to d3 site